extra/resources/SysInfo

```diff
@@ -149,11 +149,7 @@ UpdateStat() {
     name=$1; shift
     value="$*"
     printf "%s:\t%s\n" "$name" "$value"
-    if [ "$__OCF_ACTION" = "start" ] ; then
-        ${HA_SBIN_DIR}/attrd_updater ${OCF_RESKEY_delay} -S status -n $name -B "$value"
-    else 
-        ${HA_SBIN_DIR}/attrd_updater ${OCF_RESKEY_delay} -S status -n $name -v "$value"
-    fi
+    ${HA_SBIN_DIR}/attrd_updater ${OCF_RESKEY_delay} -S status -n $name -B "$value"
```

Comment:
Actually this is not to detect legacy attrd, but rather to avoid the chance of delay-flapping. We only set the delay when the resource starts, to avoid a possible collision between the monitor and some external script repeatedly changing the delay.

I think a better approach would be to silently map -B to -v in legacy attrd (ie set the value and ignore the delay)